diff --git a/Kconfig.sources b/Kconfig.sources
index ceaddc267dbb..b1780c5bc431 100644
--- a/Kconfig.sources
+++ b/Kconfig.sources
@@ -2,13 +2,13 @@
 source "$BACKPORT_DIR/compat/Kconfig"
 
 # these are copied from the kernel
-source "$BACKPORT_DIR/drivers/dma-buf/Kconfig"
+#source "$BACKPORT_DIR/drivers/dma-buf/Kconfig"
 
-source "$BACKPORT_DIR/drivers/gpu/drm/Kconfig"
+#source "$BACKPORT_DIR/drivers/gpu/drm/Kconfig"
 source "$BACKPORT_DIR/drivers/gpu/drm/i915/Kconfig"
-source "$BACKPORT_DIR/drivers/video/Kconfig"
+#source "$BACKPORT_DIR/drivers/video/Kconfig"
 
-source "$BACKPORT_DIR/drivers/char/agp/Kconfig"
+#source "$BACKPORT_DIR/drivers/char/agp/Kconfig"
 
 #source "$BACKPORT_DIR/net/bluetooth/Kconfig"
 #source "$BACKPORT_DIR/drivers/net/wireless/Kconfig"
diff --git a/local-symbols b/local-symbols
index b53b143060fb..e1d2fc00299e 100644
--- a/local-symbols
+++ b/local-symbols
@@ -46,7 +46,6 @@ DRM_KUNIT_TEST=
 DRM_KMS_HELPER=
 DRM_DEBUG_DP_MST_TOPOLOGY_REFS=
 DRM_DEBUG_MODESET_LOCK=
-DRM_FBDEV_EMULATION=
 DRM_FBDEV_OVERALLOC=
 DRM_FBDEV_LEAK_PHYS_SMEM=
 DRM_LOAD_EDID_FIRMWARE=
